bin/generate-mailing-list-us-only.rb

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env ruby
```

Comment:
before @philbogle chimes in ;)

add a comment here with what this file does? Copypasting the roxanne spec is probably good.